application/src/main/java/org/togetherjava/tjbot/features/basic/MemberCounter.java

```diff
@@ -0,0 +1,39 @@
+package org.togetherjava.tjbot.features.basic;
+
+import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.entities.Guild;
+
+import org.togetherjava.tjbot.config.Config;
+import org.togetherjava.tjbot.features.Routine;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class MemberCounter implements Routine {
+    private final String wordsInCategory;
+
+    public MemberCounter(Config config) {
+        this.wordsInCategory = config.getMemberCountCategoryName();
+    }
+
+    @Override
+    public Schedule createSchedule() {
+        return new Schedule(ScheduleMode.FIXED_RATE, 0, 12, TimeUnit.HOURS);
+    }
+
+    @Override
+    public void runRoutine(JDA jda) {
+        jda.getGuilds()
+            .stream()
+            .map(Guild::getCategories)
+            .flatMap(List::stream)
+            .filter(cat -> cat.getName().contains(wordsInCategory))
```

Comment:
`cat` could be null so you need to assert that i.e. `.filter(cat -> wordsInCategory != null && cat.getName().contains(wordsInCategory))`